exdb/models.py

```diff
@@ -59,25 +59,36 @@ class Experience(models.Model):
         ('f', _('Floor')),
     )
 
+    FUND_TYPES = (
+        ('na', _('Not necessary')),
+        ('y', _('Yes, but request not submitted yet')),
```

Comment:
I don't care either way for this one. Anyone else feel that it should be changed?